### ISSUE TITLE
to_undirected() for Edge Attributes?

### ISSUE THREAD

**Issue by reshinthadithyan** [rusty1s/pytorch_geometric#1427]
## 🐛 Bug

Edge_Index has method **to_undirected()**, is there any method by which this is extended to Edge_Attributes too? Say, if I have another Tensor which has the attributes to the corresponding indices in Edge_Index what will happen?    
## Question
Should **to_undirected()**, be extended to edge_attributes too(if available)?

**Comment by rusty1s** [comment]
That's not really possible since we cannot make any assumptions about reversed edge features. I think in this case, its best to implement the desired behaviour by yourself. For example, in case you want to add the same edge features for reverse edges, one could do:
```python
row, col = edge_index
edge_index = torch.stack([torch.cat([row, col]), torch.cat([col, row])], dim=0)
edge_attr = torch.cat([edge_attr, edge_attr], dim=0)
```

**Comment by reshinthadithyan** [comment]
@rusty1s Sorry, forgot to seed the code, issue's from my side. Due to that, I observed that **_to_undirected()_** shuffling Edge_Attr. Just now checked the Source Code. Thanks for the clarification.
